### PR TITLE
Expand the Gift Options preferences

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) expands the Gift Options preferences

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/sales/gift-options.html

### CODE REVIEW

File: src/sales/gift-options.md [hunk 1]
```diff
@@ -26,13 +26,13 @@ The Gift Options configuration applies to the entire website, but can be overrid
...
-   - **Allow Gift Messages for Order Items**
+   - For **Allow Gift Messages on Order Level**, select `Yes` to enable a single gift message for the whole order.
+   - For **Allow Gift Messages for Order Items**, select `Yes` to enable adding gift messages separately to individual items in the customer shopping cart.
 
 1. {:.ee-only}Set the gift wrapping options according to your preference:
 
-   - **Allow Gift Wrapping on Order Level**
-   - **Allow Gift Wrapping for Order Items**
+   - For **Allow Gift Wrapping on Order Level**, select `Yes`to enable a single gift wrapping for the entire order.
+   - For **Allow Gift Wrapping for Order Items** select `Yes` to enable adding gift wrapping individually to each item in the customer shopping cart.
```
Comment (leslietilling) [inline src/sales/gift-options.md:35]:
```suggestion
   - For **Allow Gift Wrapping for Order Items**, select `Yes` to enable adding gift wrapping individually to each item in the customer shopping cart.
```

File: src/sales/gift-options.md [hunk 2]
```diff
@@ -26,13 +26,13 @@ The Gift Options configuration applies to the entire website, but can be overrid
 
 1. Set the gift message options according to your preference:
 
-   - **Allow Gift Messages on Order Level**
-   - **Allow Gift Messages for Order Items**
+   - For **Allow Gift Messages on Order Level**, select `Yes` to enable a single gift message for the whole order.
+   - For **Allow Gift Messages for Order Items**, select `Yes` to enable adding gift messages separately to individual items in the customer shopping cart.
```
Comment (leslietilling) [inline src/sales/gift-options.md:30]:
```suggestion
   - For **Allow Gift Messages for Order Items**, select `Yes` to enable adding separate gift messages for individual items in the customer shopping cart.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just a couple of minor tweaks.

File: src/sales/gift-options.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/sales/gift-options.md]: Hi @leslietilling , thank you for reviewing. Done ✅

File: src/sales/gift-options.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/sales/gift-options.md]: Done ✅

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**